### PR TITLE
Add definition for 'transfeminine'

### PR DESCRIPTION
**Pull request by excitedleigh**
I've more or less copied the structure of **non-binary**, but I wasn't sure if I should open with an "of, or relating to"-style phrase, or how to best phrase it if so.

Transmasculine could _probably_ be defined similarly, substituting -feminine for -masculine and female for male, but I didn't add it as it's not part of my own identity.

I've also left the **-femme** suffix in under **trans**, I think "transfemme" and "transfeminine" are equivalent, but I'm not sure and I mostly hear and use the latter.

### CODE REVIEW

File: 11ty/definitions/transfeminine.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Transfeminine
+slug: transfeminine
+speech: adj
+defined: true
+---
+
+a [transgender](/definitions/transgender) person for whom femininity forms part of their gender.
```
Comment (tatianamac) [inline 11ty/definitions/transfeminine.md:8]:
```suggestion
describes a [transgender](/definitions/transgender) person for whom femininity forms part of their gender.
```

File: 11ty/definitions/transfeminine.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Transfeminine
+slug: transfeminine
+speech: adj
+defined: true
+---
```
Comment (tatianamac) [inline 11ty/definitions/transfeminine.md:6]:
```suggestion
reading:
 - text: What does transmasculine and transfeminine mean
   href: http://allaboutgender.com/2018/07/08/what-does-transmasculine-and-transfeminine-mean/
---
```

**Comment by tatianamac** [comment]
@excitedleigh Thank you so much for submitting this definition! I added a suggestion to get around the fact that this is an adjective (and the definition made it seem more like a noun imho).

I also added one suggestion for a link. I try to ensure each definition has "further reading" so that people who are curious don't "dead end" here. Open if you want to suggest a different link or additional links.

Lastly, once this is merged, would you be okay with me thanking you on the Self-Defined Twitter account? If yes, then what handle shall I tag?

File: 11ty/definitions/transfeminine.md (hunk 2 above)
Comment (excitedleigh) [reply 11ty/definitions/transfeminine.md:6]: 🤔 That link looks okay at first pass, although it does use "transman" and "transwoman" (as opposed to "trans man" and "trans woman") which is a minor red flag. I'll see if I can find something that's written by a transfeminine (or transmasculine) person themselves, but if I can't this is good.

**Comment by excitedleigh** [comment]
Responded inline, cheers :)

Happy to have a shout-out, I'm @ExcitedLeigh on twitter as well!

File: 11ty/definitions/transfeminine.md (hunk 2 above)
Comment (tatianamac) [reply 11ty/definitions/transfeminine.md:6]: Ah, I did not know that as a red flag, and now I do, thank you for educating me! I'm going to merge this in as I think it is okay for it to have no further readings for now (and would much prefer it have no harmful further readings!).

If you have time/energy/desire to find a better resource, I invite you to open a new PR! 💙

**Pull request merged**